h/static/scripts/annotator/monkey.coffee

```diff
@@ -9,6 +9,12 @@ Util = Annotator.Util
 delete Annotator.prototype.events[".annotator-hl mouseover"]
 delete Annotator.prototype.events[".annotator-hl mouseout"]
 
+# Add createHighlight
+Annotator.prototype.createHighlight = ->
```

Comment:
> What is the purpose of adding this here, as opposed to guest.coffee ?

I am asking this because we have all our highlighting-related code in guest.

And more generally, all the features that are outside of the scope of upstream annotator live in guest. In the monkey patch, we have the code that changes/improves the internal workings of Annotator, independently of it's Hypothes.is implementation ... and the highlights don't seem to fit this definition.